demo/app/components/table/table.component.ts

```diff
@@ -88,63 +89,72 @@ export class ExampleHttpDao {
   templateUrl: './table.component.html',
   styleUrls: ['./table.component.scss'],
 })
-export class TableComponent implements AfterViewInit {
+export class TableComponent implements OnInit, AfterViewInit, OnDestroy {
   private readonly columnsSource: CustomColumn[] = [
     {
       display: 'Title',
       name: 'title',
-      width: '400px',
+      width: 200,
       control: new FormControl(true),
     },
     {
       display: 'Comments',
       name: 'comments',
-      control: new FormControl(true),
+      width: 100,
+      control: new FormControl(false),
     },
     {
       display: 'Number',
       name: 'number',
+      width: 100,
       control: new FormControl(true),
     },
     {
       display: 'Updated',
       name: 'updated',
+      width: 100,
       control: new FormControl(true),
     },
     {
       display: 'State',
       name: 'state',
-      control: new FormControl(true),
+      width: 100,
+      control: new FormControl(false),
     },
     {
       display: 'Labels',
       name: 'labels',
+      width: 200,
       control: new FormControl(true),
     },
-    {
-      display: 'Body',
-      name: 'body',
-      width: '260px',
-      control: new FormControl(true),
-    },
+    // {
+    //   display: 'Body',
+    //   name: 'body',
+    //   width: '260px',
+    //   control: new FormControl(true),
+    // },
```

Comment:
I have been leaving it in since it's the only column that has really long content. So if we need to check how really long content effects columns etc we can just uncomment without needing to go see what the GitHub API can return.